nest-forms-backend/src/ginis/ginis.service.ts

```diff
@@ -524,8 +544,8 @@ export default class GinisService {
     ) {
       if (
         !form.ginisDocumentId ||
-        !form.schemaVersion.ginisOrganizationName ||
-        !form.schemaVersion.ginisPersonName
+        !formDefinition.ginisAssignment?.ginisOrganizationName ||
```

Comment:
`ginisAssignment` is optional in `FormDefinitionSlovenskoSkBase`, should it be? Will this code fail if it is not provided? Shouldn't we make it required?

Reply:
I think it should be optional, because for example Tax forms do not go through Ginis, so they don't have this data.

Reply:
Does it make sense to move it to `FormDefinitionSlovenskoSkGeneric` then and make it required?

Reply:
Yep this seems as a good idea. I can move it.